Makefile

```diff
@@ -24,5 +24,9 @@ nose3: qless-core
 	rm -rf .coverage
 	nosetests3 --exe --cover-package=qless --with-coverage --cover-branches -v
 
+.PHONY: requirements
+requirements:
```

Comment:
You could simply make the target `requirements.txt`. Technically, that's how make is supposed to work. Then it wouldn't need to be a `.PHONY` target.